packages/account-v2/src/containers/PersonalDetailsForm/__tests__/SupportProfessionalClient.spec.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { Form, Formik } from 'formik';
+import { APIProvider, AuthProvider } from '@deriv/api-v2';
+import { render, screen } from '@testing-library/react';
+import { usePersonalDetails } from '../../../hooks/usePersonalDetails';
+import { SupportProfessionalClient } from '../SupportProfessionalClient';
+
+jest.mock('../../../hooks/usePersonalDetails', () => ({
+    ...jest.requireActual('../../../hooks/usePersonalDetails'),
+    usePersonalDetails: jest.fn(),
+}));
+
+beforeEach(() => {
+    (usePersonalDetails as jest.Mock).mockReturnValue({
+        accountAuthStatus: { isAccountVerified: false },
+        data: { isVirtual: false },
+    });
+});
+
+describe('PersonalDetails', () => {
+    const renderComponent = (children: React.ReactNode) => {
+        render(
+            <APIProvider>
+                <AuthProvider>
+                    <Formik initialValues={{}} onSubmit={jest.fn()}>
+                        <Form>{children}</Form>
+                    </Formik>
+                </AuthProvider>
+            </APIProvider>
+        );
+    };
+
+    const description1 =
+        'By default, all Deriv.com clients are retail clients but anyone can request to be treated as a professional client.';
+    const description2 = 'A professional client receives a lower degree of client protection due to the following.';
+    const description3 =
+        'We presume that you possess the experience, knowledge, and expertise to make your own investment decisions and properly assess the risk involved.';
+    const description4 =
+        'We’re not obliged to conduct an appropriateness test, nor provide you with any risk warnings.';
+
```

Comment:
cant we import  packages/account-v2/src/constants/supportProfessionalClientConstants.ts and test this